sdk/core/azure-core/inc/azure/core/exception.hpp

```diff
@@ -65,7 +65,7 @@ namespace Azure { namespace Core {
     std::unique_ptr<Azure::Core::Http::RawResponse> RawResponse;
 
     /**
-     * @brief Constructs a new `RequestFailedException` object.
+     * @brief Constructs a new `%RequestFailedException` with a message string.
```

Comment:
Consistency nit (consider fixing elsewhere too):
```suggestion
     * @brief Constructs a new `%RequestFailedException` with a \p message string.
```

Reply:
Will take to the follow up PR.

Reply:
#2390